packages/cactus-plugin-ledger-connector-iroha/package.json

```diff
@@ -63,6 +63,7 @@
     "axios": "1.5.1",
     "express": "4.18.2",
     "fast-safe-stringify": "2.1.1",
+    "grpc": "1.24.11",
```

Comment:
Without it the sample app can't be started, will remove it just before merge (kept for now to simplify manual testing)

Reply:
Removed